docs/vscode.md

```diff
@@ -1,37 +1,27 @@
+Visual Studio Code is a commonly used editor by workerd developers (other editors are great too!). These notes present some getting started tips. If you have tricks and tips that would improve the developer experience, please let us know!
 
-## clangd
+## Recommended extensions for developing workerd
 
-We use [clangd](https://marketplace.visualstudio.com/items?itemName=llvm-vs-code-extensions.vscode-clangd)
-for code completion and diagnostics.
+The recommended extensions to install are:
 
-For debugging, you will need the [Microsoft C/C++ extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) installed.
+* [LLVM clangd extension](https://marketplace.visualstudio.com/items?itemName=llvm-vs-code-extensions.vscode-clangd)
+for code completion and navigation.
 
-The Microsoft C/C++ extension has IntelliSense support that is not compatible with the clangd extension. We recommend disabling the
-Microsoft IntelliSense Engine for this project (`Settings -> C_Cpp.intelliSenseEngine -> disabled`).
+  For clangd to work well, you need to generate a `compile_commands.json` file. This is described below in [Generating compile_commands.json](#generating-compile_commandsjson).
 
-For clangd to work well, you need to generate a `compile_commands.json` file. This can be done from within VSCode using
-`Run Task -> Generate compile_commands.json` or at the command-line:
+* [Microsoft C/C++ extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) for debugging, syntax highlighting, etc.
 
-```sh
-# Build workerd to ensure all generated sources are present.
-$ bazel build //...
-$ bazel run //:refresh_compile_commands
-```
+  The Microsoft C/C++ extension has IntelliSense support that is not compatible with the clangd extension. We recommend disabling the Microsoft IntelliSense Engine for this project ("Settings → C_Cpp.intelliSenseEngine → disabled").
 
-If this command fails, try again after running:
+* [Capnproto-syntax extension](https://marketplace.visualstudio.com/items?itemName=abronan.capnproto-syntax) for syntax highlighting if you are editing `.capnp` files.
 
-```sh
-$ bazel clean --expunge
-$ bazel test //...
-```
+* [GitLens extension](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens) for super charged git functionality within Visual Studio Code.
 
-There is an issue between workerd's bazel setup and Hedron's compile_commands.json generator (tracked in
-https://github.com/cloudflare/workerd/issues/506).
+You can install all of these extensions with the **Extensions: Configure Recommended Extensions (Workspace Folder)** command. You can do find this through the Visual Studio Code Command Palette (`shift+ctrl+p` on Linux / Windows, `shift+cmd+p` on OS X) and typing "Configure Recommended Extensions". The recommendations that will be installed can be found in the [.vscode/extensions.json](../.vscode/extensions.json) file.
```

Comment:
[nit] `You can do` -> `You can`

Reply:
Thanks, experimenting with incorporating this comment in vscode...
```suggestion
You can install all of these extensions with the **Extensions: Configure Recommended Extensions (Workspace Folder)** command. You can find this through the Visual Studio Code Command Palette (`shift+ctrl+p` on Linux / Windows, `shift+cmd+p` on OS X) and typing "Configure Recommended Extensions". The recommendations that will be installed can be found in the [.vscode/extensions.json](../.vscode/extensions.json) file.
```

Reply:
Thanks incorporated in https://github.com/cloudflare/workerd/pull/610/commits/60af28eff87c7ce4efed0cb08eb5a81366adfc70